### DISCUSSION TITLE
`prisma migrate deploy` problems on production environment

### DISCUSSION THREAD

**Discussion by Frostbourn** [prisma/prisma#17716] [Q&A]
### Bug description

Hi,

Im trying to deploy my project built with Next.js to prod enviroment on k8s but when I run first command `prisma migrate deploy` I get the following error:

`Error: Column count of mysql.proc is wrong. Expected 21, found 20. Created with MariaDB 100128, now running 100424. Please use mysql_upgrade to fix this error`

Unfortunatelly I don't have acces to the mysql server. Is there any possibility to fix this issue?

### How to reproduce

I cant give any details because its deployed by our company's helm chart.

### Prisma information

```prisma
model Page {
  id Int @id @default(autoincrement())

  createdAt     DateTime @default(now())
  updatedAt     DateTime @updatedAt
  name          String   @db.VarChar(255)
  url           String   @db.VarChar(255)
  published Boolean @default(false)

  @@unique(url)
  @@index(url)
}

model Confirmation {
  id             Int      @id @default(autoincrement())
  createdAt      DateTime @default(now())
  updatedAt      DateTime @updatedAt
  confirmed      Boolean  @default(false)
  email          String   @db.VarChar(255)
}
```

### Environment & setup

- OS: Debian
- Database: MariaDB 10.4.24
- Node.js version: 18.12.1

### Prisma Version

4.8.0

**Comment by janpio** [comment]
That seems to be an error message from your database server, that is triggered when some of the migration SQL is run. 
I don't expect that to be something that Prisma can fix in any way.

Is there anything special about the SQL that it is trying to migrate? Can you maybe share it?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
